Microsoft.Toolkit.Services/Microsoft.Toolkit.Services.csproj

```diff
@@ -32,8 +36,36 @@
     <ProjectReference Include="..\Microsoft.Toolkit.Uwp\Microsoft.Toolkit.Uwp.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="!('$(TargetFramework)'=='net461')">
+  <ItemGroup Condition="!('$(TargetFramework)'=='net462')">
     <Compile Remove="Services\MicrosoftGraph\WinForms\**\*" />
+    <Compile Remove="PlatformSpecific\NetFramework\**\*" />
+    <None Remove="PlatformSpecific\NetFramework\**\*" />
+    <Page Remove="PlatformSpecific\NetFramework\**\*" />
+  </ItemGroup>
+
+  <ItemGroup Condition="'$(TargetFramework)'=='net462'">
+    <Reference Include="System.Data" />
+    <Reference Include="System.ComponentModel" />
+    <Reference Include="System.Drawing" />
+    <Reference Include="System.Windows" />
+    <Reference Include="System.Windows.Forms" />
+    <Reference Include="System.Web" />
+    <Reference Include="System.Xaml">
+      <RequiredTargetFramework>4.0</RequiredTargetFramework>
+    </Reference>
+    <Reference Include="WindowsBase" />
+    <Reference Include="PresentationCore" />
+    <Reference Include="PresentationFramework" />
+    <PackageReference Include="Microsoft.Toolkit.Wpf.UI.Controls.WebView" Version="5.0.0-preview.gb86cb1c4cb" />
```

Comment:
This should be a ProjectReference, not a PackageReference. The way you did it, it is tied to a specific version, not generating the nuget dependencies correctly.

Reply:
We didn't reference the project as it's in a different solution, and it may be moved to a different repository all together in the future. If you think we should do it anyway, does the syntax to reference a project on the same solution differs from one on a different solution?